### PR TITLE
Upgrade to kubeedge v0.2 and enhanced scripts to autogenerate configuration

### LINKED ISSUE #13: Upgrade security example to support kubeedge v0.2
Security example needs to be upgraded to kubeedge v0.2 to support cloud components and also enhance scripts based on the cleanup.

### PR DESCRIPTION
**Pull request by trilokgm**
What type of PR is this?
/kind enhancement

What this PR does / why we need it
Build script is updated to use configuration from repos. 
Auto-generation of configuration based on env variables is supported.
Kubeedge v0.2 with cloud components is supported.

Which issue(s) this PR fixes
Fixes #13

### CODE REVIEW

File: security-demo/cloud-stub/cmd/cloud_stub.go [hunk 1]
```diff
@@ -19,6 +19,6 @@ func main() {
 
 	stub := cloudhub.NewCloudStub(config)
 
-	go stub.Start()
+	//go stub.Start()
```
Comment (m1093782566) [inline security-demo/cloud-stub/cmd/cloud_stub.go]: nit: please remove the dead line.

File: security-demo/cloud-stub/pkg/controller/cloudhub/stub.go [hunk 2]
```diff
@@ -215,7 +215,7 @@ func (tm *StubCloudHub) Start() {
 func (tm *StubCloudHub) PlacementServer() {
 	fmt.Println("started placement server")
 	router := mux.NewRouter()
-	router.HandleFunc("/pod", tm.podHandler)                                       // for pod test
+	//router.HandleFunc("/pod", tm.podHandler)                                       // for pod test
```
Comment (m1093782566) [inline security-demo/cloud-stub/pkg/controller/cloudhub/stub.go]: remove the dead line.

File: security-demo/cloud-stub/cmd/cloud_stub.go (hunk 1 above)
Comment (trilokgm) [reply security-demo/cloud-stub/cmd/cloud_stub.go]: Oops!! Accidental test code checked-in. Reverted back the changes.
Thanks for pointing out.

File: security-demo/cloud-stub/pkg/controller/cloudhub/stub.go (hunk 2 above)
Comment (trilokgm) [reply security-demo/cloud-stub/pkg/controller/cloudhub/stub.go]: Oops!! Accidental test code checked-in. Reverted back the changes.
Thanks for pointing out.

**Pull request merged**